app/views/steps/submission/declaration/_initial.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1 class="govuk-heading-xl">
```

Comment:
This is the declaration of the initial application and has not changed, just moved to a new partial.